Gemfile.lock

```diff
@@ -344,7 +344,7 @@ GEM
       metasm
       rex-core
       rex-text
-    rex-socket (0.1.26)
+    rex-socket (0.1.28)
```

Comment:
Looks like we'll want to bump this to 0.1.29 to grab in the additional changes made by https://github.com/rapid7/rex-socket/pull/31

Reply:
Bumped!